metadata-ingestion/docs/sources/neo4j/neo4j_recipe.yml

```diff
@@ -0,0 +1,14 @@
+source:
+    type: 'neo4j'
+    config:
+        uri: 'neo4j+ssc://host:7687'
+        username: 'neo4j'
+        password: 'password'
+        node_tag: 'Node'
+        relationship_tag: 'Relationship'
```

Comment:
I believe these two config options node_tag and relationship_tag can now be removed ? We need not accept subtypes as config, as they are constants.